test/conformance/kuttl/generate/clusterpolicy/cornercases/cpol-clone-sync-reinstall-policy/README.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+This is a corner case test to ensure a generate data rule can be triggered on the deletion of the trigger resource.
```

Comment:
"generate clone rule". Also add this is to ensure upgrades to 1.10 are successful, something of that nature.